AntennaTracker/Parameters.h

```diff
@@ -87,6 +87,7 @@ class Parameters {
         k_param_notify,
         k_param_BoardConfig_CAN,
         k_param_battery,
+        k_param_gcs_pid_mask,
```

Comment:
this shifts everything below it down by one which corrupts everything. Best to put it down at line 116 below k_param_command_total  with something like
`k_param_gcs_pid_mask = 225,`
so we don't change any other enum index